src/events/components/ImageView/index.tsx

```diff
@@ -46,14 +46,13 @@ const isPopulated = (imageEvents: IState) => {
 };
 
 export const ImageView = ({  }: IProps) => {
-  const { eventList, updateEventList } = useContext(EventsRepo);
+  const dispatch = useDispatch();
+  const eventList = useSelector((state) => eventSelectors.selectAll(state));
+  // const { eventList, updateEventList } = useContext(EventsRepo);
```

Comment:
Could remove this comment, won't need it ever again :hear_no_evil:

Reply:
Oops! 😅